src/common/tagging/external/tag_group.go

```diff
@@ -206,6 +213,14 @@ func (t *TagGroup) ExtractExternalGroupsTags(tagsConfig TagsConfig) []Tag {
 	return groupTags
 }
 
+func evaluateTemplateVariable(val string) string {
+	envVariableMatch := EnvVariableRegex.FindStringSubmatch(val)
```

Comment:
@tronxd What happens if the env variable is not set?

Reply:
@nimrodkor Logger warning and fallback to default value